src/main/java/com/ironhack/ironLibrary/model/Book.java

```diff
@@ -6,12 +6,18 @@
 
 @Data
 @Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "books")
 public class Book {
     @Id
     @Setter(AccessLevel.NONE)
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
i think that @GeneratedValue it's not necessary because  we gonna pass isbn so db not gonna need to create any id random.

Reply:
Hecho! Es que se había quedado de las pruebas que estuvimos haciendo el martes.